middleware/rewrite/README.md

```diff
@@ -23,6 +23,10 @@ rewritten; e.g., to rewrite CH queries to IN use `rewrite class CH IN`.
 When the FIELD is `name` the query name in the message is rewritten; this
 needs to be a full match of the name, e.g., `rewrite name miek.nl example.org`.
 
+When the FIELD is `edns0` an EDNS0_LOCAL option can be appended to the request;
```

Comment:
We can do that (`edn0 local`). FYI, Friday was @bwaz last day at Infoblox so someone else (maybe me?) will be wrapping this up.